app/validators/sections/section_validator.py

```diff
@@ -360,3 +366,23 @@ def _validate_answers(self, question):
                     answer_id=answer["id"],
                 )
             self.errors += answer_validator.errors
+
+    def _validate_multiple_list_collectors(self):
+        for_lists = []
+
+        for block_id in self.questionnaire_schema.get_section_block_ids(
+            self.section["id"]
+        ):
+            block = self.questionnaire_schema.get_block(block_id)
+            if block["type"] in ["ListCollector", "ListCollectorContent"]:
+                if block.get("summary"):
```

Comment:
why do you break if the block has a summary?

Reply:
This summary field present means section will have a list on section summary ("list summary") enabled, and we want to allow multiple list collectors when such "list summary" is enabled. From what @MebinAbraham said: "if summary without items or no summary or it is a list summary (census version) then allow multiple else disallow".

Reply:
ohhh, I thought it was list summary or summary with items that was banned but that makes sense, I see!

If you had one list with a summary, and one without, but the first one breaks out of the loop, could that break this?

Reply:
I think it would be helpful to have that explanation in a comment - that validation isn't needed for list summaries, and since the possible bug mentioned is existing before this PR, that could be a follow up too